Create v1.1.2.md (#21)
harvester/release-notes: v1.1.2.md

* Cluster field is not really \*required\* when creating RKE2/K3s Harvester [[#3165](https://github.com/harvester/harvester/issues/3165)]
* Harvester CPI Error: failed to get node modifiers from cloud provider: failed to find kubelet node IP from cloud provider, requeuing [[#3219](https://github.com/harvester/harvester/issues/3219)]
* server_url with https but without :443 causes node join to fail [[#3273](https://github.com/harvester/harvester/issues/3273)]
* Harvester Cloud Provider truncates Harvester Loadbalancer name without checking for valid fqdn [[#3333](https://github.com/harvester/harvester/issues/3333)]
* Cluster Member cannot see clusternetworks [[#3334](https://github.com/harvester/harvester/issues/3334)]
* Multi Nic on VM Node Driver [[#3335](https://github.com/harvester/harvester/issues/3335)]
* Older versions of the Harvester plugin have style compatibility issues with the latest version of Rancher [[#3373](https://github.com/harvester/harvester/issues/3373)]
* Can not create RKE2 cluster after enabling Elemental extension [[#3374](https://github.com/harvester/harvester/issues/3374)]

> * No resource limits set for the Kubevirt's pod [[#3427](https://github.com/harvester/harvester/issues/3427)]

Reviewer: ```suggestion
* No resource limits set for the Kubevirt pods in the harvester-system namespace [[#3427](https://github.com/harvester/harvester/issues/3427)]
```

* Cannot edit namespace on single cluster [[#3435](https://github.com/harvester/harvester/issues/3435)]
* The harvester plugin affects the display of the global settings page [[#3438](https://github.com/harvester/harvester/issues/3438)]

> * RKE2 stuck in creating (2 NICs) nodes with error: flags provided: missing section: network [[#3443](https://github.com/harvester/harvester/issues/3443)]

Reviewer: ```suggestion
* Provisioning RKE2 cluster with 2 NICs fails with error: flags provided: missing section: network [[#3443](https://github.com/harvester/harvester/issues/3443)]
```

> * Can't upgrade from master version to master version [[#3444](https://github.com/harvester/harvester/issues/3444)]

Reviewer: ```suggestion
* Can't upgrade from pre-build master version to same master version [[#3444](https://github.com/harvester/harvester/issues/3444)]
```

* Templates not clickable in VM creation [[#3493](https://github.com/harvester/harvester/issues/3493)]

> * Grafana monitor metric is not visible [[#3537](https://github.com/harvester/harvester/issues/3537)]

Reviewer: ```suggestion
* Grafana Cluster Metrics and VM Metrics are not visible [[#3537](https://github.com/harvester/harvester/issues/3537)]
```

# Harvester v1.1.2 Release Note

This release follows Harvester v1.1.1, the documentation is available at https://docs.harvesterhci.io/v1.1.

# Downloads

## ISO
:cd: [harvester-v1.1.2-amd64.iso](https://releases.rancher.com/harvester/v1.1.2/harvester-v1.1.2-amd64.iso)

## Additional Artifacts
:file_folder: [harvester-v1.1.2-initrd-amd64](https://releases.rancher.com/harvester/v1.1.2/harvester-v1.1.2-initrd-amd64)
:file_folder: [harvester-v1.1.2-vmlinuz-amd64](https://releases.rancher.com/harvester/v1.1.2/harvester-v1.1.2-vmlinuz-amd64)
:file_folder: [harvester-v1.1.2-rootfs-amd64.squashfs](https://releases.rancher.com/harvester/v1.1.2/harvester-v1.1.2-rootfs-amd64.squashfs)
:white_check_mark: [harvester-v1.1.2-amd64.sha512](https://releases.rancher.com/harvester/v1.1.2/harvester-v1.1.2-amd64.sha512)
:memo:  [harvester-v1.1.2-version.yaml](https://releases.rancher.com/harvester/v1.1.2/version.yaml)

# Install
Harvester can be installed with the [ISO](https://docs.harvesterhci.io/v1.1/install/iso-install/) image or via [PXE](https://docs.harvesterhci.io/v1.1/install/pxe-boot-install/) boot.

# Highlights

> * terraform-provider-rancher2 to support to add additional NIC and disks [[#2733](https://github.com/harvester/harvester/issues/2733)]

Reviewer: ```suggestion
* terraform-provider-rancher2 supports adding multiple NIC and disks [[#2733](https://github.com/harvester/harvester/issues/2733)]
```

* Bump baseOS to SLE Micro for Rancher 5.3 [[#3280](https://github.com/harvester/harvester/issues/3280)]
* Bump Harvester v1.1.2 dependencies [[#3321](https://github.com/harvester/harvester/issues/3321)]

# Known Issues
* Volume created by the Harvester CSI driver gets deleted after reprovisioning RKE2 node [[#3272](https://github.com/harvester/harvester/issues/3272)] [[Workaround](https://github.com/harvester/harvester/issues/3272#issuecomment-1459815299)]
* Upgrade
  * Upgrade stop at upgrading node3 stage, stuck in Pre-drained state [[#3021](https://github.com/harvester/harvester/issues/3021)] [[Workaround](https://github.com/harvester/harvester/issues/3021#issuecomment-1288747614)]
  * Upgrade stuck in upgrading first node: Job was active longer than specified deadline [[#2894](https://github.com/harvester/harvester/issues/2894)] [[Workaround](https://github.com/harvester/harvester/issues/2894#issuecomment-1274069690)]
  * After upgrading from v1.0.3 to v1.1.1, when you add a node it doesn't assign a cluster network [[#3223](https://github.com/harvester/harvester/issues/3223)] [[Workaround](https://github.com/harvester/harvester/issues/3223#issuecomment-1331854544)]

>   * v1.1.1 upgrade to v1.1.2-rc3 stuck in upgrading system service [[#3675](https://github.com/harvester/harvester/issues/3675)]

Reviewer: ```suggestion
  * Upgrade from v1.1.1 to v1.1.2-rc3 stuck in upgrading system service [[#3675](https://github.com/harvester/harvester/issues/3675)]
```

# Harvester v1.1.2 Release Note

This release follows Harvester v1.1.1, the documentation is available at https://docs.harvesterhci.io/v1.1.

# Important information about Rancher support

> If you use Rancher v2.7.2 with Harvester v1.1.1 clusters, you can't select the Harvester cloud provider while deploying guest clusters. See [this issue](https://github.com/harvester/harvester/issues/3750) for more information.

Reviewer 2: ```suggestion
If you are using Rancher v2.7.2 with Harvester v1.1.1 clusters, you will not be able to select the Harvester cloud provider when deploying/updating guest clusters. This issue has been documented in detail in [#3750](https://github.com/harvester/harvester/issues/3750).
```

> There are two fixes to this issue:
> - Upgrade Harvester to v1.1.2.
> - Manually patch the `harvester-csi-ccm-versions` setting:

Reviewer 2: ```suggestion
To resolve this issue, there are two options available:
- Upgrade your Harvester cluster to v1.1.2.
- Apply a manual patch to the `harvester-csi-ccm-versions` setting:
```
